### ISSUE TITLE
 Unknown activation function none？

### ISSUE THREAD

**Issue by ngunauj** [mapillary/inplace_abn#122]
Is it necessary to use the activation function when use InPlaceABNSync?
Sometimes I just want to use bn without activation function,How can I set it up?

**Comment by ngunauj** [comment]
may be i can use 'Identity' inplace  no activation function...

**Comment by ducksoup** [comment]
@ngunauj exactly, the correct setting when you don't need an activation function is `activation="identity"`!

**Comment by ZHUANGHP** [comment]
> @ngunauj exactly, the correct setting when you don't need an activation function is `activation="identity"`!

I wonder if using the identity activation leads to the same result as the nn.BatchNorm2d?
